code/modules/client/preferences_spawnpoints.dm

```diff
@@ -66,17 +66,6 @@ GLOBAL_VAR(spawntypes)
 	..()
 	turfs = GLOB.latejoin_cryo
 
-/datum/spawnpoint/cryo/after_join(mob/living/carbon/human/victim)
```

Comment:
You're going to also need to make this change to cryo2 below. However, I would prefer them to still wake up in the cryopod, even without the sleeping debuff.